server/config.go

```diff
@@ -67,7 +67,13 @@ func ParseConfig(bytes []byte) (*Config, error) {
 	}
 
 	c.Github.SetValuesFromEnv("")
-	c.Server.SetValuesFromEnv("BULLDOZER_")
+
+	serverPrefix := "BULLDOZER_"
```

Comment:
As a style note, I'd like to call this `envPrefix` and make the default value a constant at the top of this file:

```go
const (
    DefaultEnvPrefix = "BULLDOZER_"
)
```